tools/sadguess

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-WORKDIR=`pwd`
-BASISDIR='../basis'
+WORKDIR=$QUICK_INSTALL/tools
```

Comment:
This line seems superfluous. We also don't have a tools directory in the installation directory afaik.